Packs/CortexAttackSurfaceManagement/README.md

```diff
@@ -1,4 +1,4 @@
-The Cortex Attack Surface Management Pack provides content for working with Attack Surface Management (ASM) data. 
+The **Cortex Attack Surface Management** Pack provides content for working with Attack Surface Management (ASM) data. 
 
 ## What does this pack do?
-- Includes playbooks for enrichment and response to alerts detected impacting your external attack surface.
+- Includes playbooks for enrichment and response to alerts detected impacting your external attack surface.
```

Comment:
Please add a short sentence describing the integration - which is also part of this pack.

Reply:
@ShirleyDenkberg added information i commit [Update README.md](https://github.com/demisto/content/pull/21226/commits/d7fa5ef72b621eac2628c3fc4423ff31ea1ca8a5)

Good enough?

Reply:
Indeed :)